### PR TITLE
Refactored app to include redux

### PR DESCRIPTION
**Pull request by sophiegraham**
-state was moved from markdown container into redux store
-added travis file

### CODE REVIEW

File: src/containers/markdown/MarkdownContainer.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  handleChange = ({ target }) => {
+    store.dispatch(updateMarkdown(target.value));
+  };
+
+  render() {
+    const { markdown } = this.state;
+    return (
+    <>
```
Comment (MariahAdams) [inline src/containers/markdown/MarkdownContainer.js]: I would indent this block of jsx.

File: src/selectors/markdownSelectors.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { markdown } from '../selectors/markdownSelectors';
+
+describe.skip('markdown selectors', () => {
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.test.js]: why is this skipped?

File: src/selectors/markdownSelectors.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const getAllMarkdown = state => {
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.js]: this only gets one markdown file. Maybe call this `getMarkdown`

File: src/selectors/markdownSelectors.test.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { markdown } from '../selectors/markdownSelectors';
```
Comment (mehtaphysical) [inline src/selectors/markdownSelectors.test.js]: fix this import

File: src/selectors/markdownSelectors.test.js (hunk 2 above)
Comment (tylercorbett) [reply src/selectors/markdownSelectors.test.js]: "We chose to focus elsewhere." - Marty Nelson, 2019.

File: src/containers/markdown/MarkdownContainer.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+export default class MarkdownContainer extends PureComponent {
+  state = {
+    markdown: '',
+    unsubscribe: null
+  };
+
+  updateState = () => {
+    const currentReduxState = store.getState();
+    const markdown = getAllMarkdown(currentReduxState);
```
Comment (mehtaphysical) [inline src/containers/markdown/MarkdownContainer.js]: fix this selector name

**Comment by tylercorbett** [comment]
Refactored based on comments

**Review by mehtaphysical — APPROVED** [review]

**Pull request merged**